modules/core/src/lib/init.js

```diff
@@ -43,5 +47,8 @@ if (!global.deck) {
     log
   };
 
+  HTMLImageLoader.extensions = ['png', 'jpg', 'gif', 'webp', 'bmp', 'svg'];
```

Comment:
Should this change be done in loaders.gl?

Reply:
https://github.com/uber-web/loaders.gl/pull/276